content/blog/2020/2020-4-015-covid-19-part-3.md

```diff
@@ -0,0 +1,114 @@
++++
+author = "Andreja Kovačič"
+date = "2020-04-13"
+draft = false
+title = "Data Mining COVID-19 Epidemics: Part 3"
+type = "blog"
+thumbImage = "/blog_img/2020/2020-04-15-thumb-log-growth.png"
+frontPageImage = "/blog_img/2020/2020-04-15-thumb-log-growth.png"
+categories = ["covid-19", "visualization", "addons", "trends", "time"]
+shortExcerpt = "Inspection of Covid-19 time trends"
+longExcerpt = "Inspecting and comparing Covid-19 time trends, absolute and relative growth."
++++
+
+# Data Mininig COVID-19 Epidemics: Part 3
+
+So far, we've seen how to do [basic visualisations](https://orange.biolab.si/blog/2020/2020-04-02-covid-19-basic/) related to the corona virus and how to look at the [disease progression on the map](https://orange.biolab.si/blog/2020/2020-04-09-covid-19-part-2/). Be sure to check that out first, before delving into this one. 
+
+We are now heading towards somehwat more advanced visualisations, that let us observe trends in the data. Just as a heads up: your results may be somewhat different, depending on the day you downloaded the data. We are working with confirmed cases up to 13.4, on previously mentioned data from John Hopkins University.
+
+## Prerequisite: Timeseries add-on
+The spread of the virus is influenced by many factors, time being one of them. Timeseries add-on specializes in manipulation of time-related data. If you followed our blogs, you should already have it installed. If you don't just take a peek in the [second one](https://orange.biolab.si/blog/2020/2020-04-09-covid-19-part-2/).
+
+## Formatting the data As Timeseries
+We'll load our data as per usual with the File widget and then select some countries we are interested in with the help of Data Table. You could also do it straight after inspecting the data in Geo Map.
+Our data contains lattitude and longitude columns, which are just getting in the way in our further analysis. We will put them into meta columns and thus excluding them from any calculations.
+Now we just need to tell Orange where our time component is. We'll use Transpose widget to make each row represent a data point and then connect it to As Timeseries Widget.
```

Comment:
where our time component is -> where is our time component

Better, we need to tell Orange which variable contains time stamps.

Reply:
Really? I would say *where our time component is*. The second version sounds too much like a question to me. But again, this is just an opinion of a non-expert.

P.s. Found [this](https://english.stackexchange.com/questions/61714/why-do-we-put-the-verb-to-be-at-the-end-of-these-questions). Seems like the first option is ok.

Reply:
Interesting. I always thought that putting the verb at the end is bad. And the this is even not a matter of discussion, unlike, say, splitting the infinitive. Thanks!

I still vote for something like "to tell Orange, which variable contains time stamps".